install_config/storage_examples/ceph_rbd_dynamic_example.adoc

```diff
@@ -0,0 +1,327 @@
+[[install-config-storage-examples-ceph-rbd-dynamic-example]]
+= Complete Example Using Ceph RBD for Dynamic Provisioning
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+This topic provides an end-to-end example of using an existing Ceph cluster as
+an {product-title} dynamic persistent store. It is assumed that a working Ceph cluster
+is already set up. If not, consult the
+link:https://access.redhat.com/products/red-hat-ceph-storage[Overview of Red Hat
+Ceph Storage].
+
+xref:../persistent_storage/persistent_storage_ceph_rbd.adoc#install-config-persistent-storage-persistent-storage-ceph-rbd[Persistent Storage
+Using Ceph Rados Block Device] provides an explanation of persistent volumes
+(PVs), persistent volume claims (PVCs), and using Ceph RBD as persistent
+storage.
+
+[NOTE]
+====
+All `oc ...` commands are executed on the {product-title} master host.
+====
+
+[[ceph-rbd-dynamic-example-installing-the-ceph-common-package]]
+== Installing the ceph-common Package
+
+The *ceph-common* library must be installed on *all schedulable* {product-title} nodes:
+
+[NOTE]
+====
+The {product-title} all-in-one host is not often used to run pod workloads and,
+thus, is not included as a schedulable node.
+====
+
+----
+# yum install -y ceph-common
+----
+
+[[ceph-rbd-dynamic-example-create-pool-for-dynamic-volumes]]
+== Create pool for dynamic volumes
+
+It is recommended to create a pool for your dynamic volumes to live.
+
+[NOTE]
+====
+Using the default pool of rbd is an option, but not recommended. 
+====
+
+From a Ceph admin or Mon node, create a new pool for dynamic volumes
+----
+ceph osd pool create kube 1024
+ceph auth get-or-create client.kube mon 'allow r' osd 'allow class-read object_prefix rbd_children, allow rwx pool=kube' -o ceph.client.kube.keyring
+----
+
+[[ceph-rbd-dynamic-example-creating-the-ceph-secret]]
+== Creating the Ceph Secret
+
+The `ceph auth get-key` command is run on a Ceph *MON* node to display the key
+value for the *client.admin* user:
+
+.Ceph Secret Definition
+====
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ceph-secret
+  namespace: kube-system
+data:
+  key: QVFBOFF2SlZheUJQRVJBQWgvS2cwT1laQUhPQno3akZwekxxdGc9PQ== <1>
+type:
+  kubernetes.io/rbd <2>
+
+----
+<1> This base64 key is generated on one of the Ceph MON nodes using the `ceph auth get-key client.admin | base64` command, then copying the output and pasting it as the secret key's value.
+<2> This is required for Ceph RBD to work with dynamic provisioning.
+====
+
+Save the secret definition to a file, for example *_ceph-secret.yaml_*,
+then create the secret:
+
+====
```

Comment:
I think you can get rid of the ====s around the code blocks and just keep the ----s. We are moving away from using those in code blocks